### ISSUE TITLE
No airport name locates it on lat/lon 0

### ISSUE THREAD

**Issue by bermi** [cristobal-io/aviation-front-end#18]
Description: On lat/lon 0 there are many airports one on top of the other
Steps to reproduce: Move the map to the African coast on lat/lon 0 there are some airports that shouldn't be there.

<img width="854" alt="screen shot 2016-10-16 at 01 49 06" src="https://cloud.githubusercontent.com/assets/27073/19414112/4639b64c-9343-11e6-94e8-637832da6783.png">
<img width="1185" alt="screen shot 2016-10-16 at 01 47 08" src="https://cloud.githubusercontent.com/assets/27073/19414115/4bccda26-9343-11e6-932c-1b41e8808960.png">

This might be an issue with the aviation model or the data.

**Comment by cristobal-io** [comment]
I've added a function that filters the airports that have coordinates value of 0, this prevents the airports from showing on map, but it doesn't fix the issue.

The issue is coming from the [aviation-scrapper](https://github.com/cristobal-io/aviation-scraper) module.

**Comment by bermi** [comment]
Did you see my ticket on aviation data?

The scrapper catches all the pages but with different airport names. Isn't the aviation data where things are normalized and grouped?

Bermi

**Comment by cristobal-io** [comment]
Yes, I saw it, you are right, aviation-json is where things are normalized and grouped, but the problem when the coordinates are not available,  might be two options:

1. the coordinates are not available on Wikipedia (I doubt it)
2. the scrapper is missing the coordinates.

So I think the problem for airports with 0,0 coordinates is on the aviation-scrapper, not on the aviation-json package.

**Comment by bermi** [comment]
Doesn't the wikipedia API return the coordinates as well? Probably only the
redirected pages are missing the location location which is something
tracked on https://github.com/cristobal-io/aviation-front-end/issues/8
